src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -64,12 +64,10 @@ public function __construct(array $server = [], History $history = null, CookieJ
 
     /**
      * Sets whether to automatically follow redirects or not.
-     *
-     * @param bool $followRedirect Whether to follow redirects
      */
-    public function followRedirects($followRedirect = true)
+    public function followRedirects(bool $followRedirect = true)
```

Comment:
Could we rename the variable Name to plural, too?
Or is it not possible?

Reply:
yes we can